lemur/tests/helper_random.py

```diff
@@ -66,7 +66,7 @@ def rand_experiment_id():
 
 
 # generate a random value for classTime attribute in Class class
-def rand_classtime(classtime_list=['16fall', '17spring', '17fall']):
+def rand_classtime(classtime_list=['FALL2016', 'SPRING2017', 'FALL2017']):
```

Comment:
Shouldn't be an issue here, because you never mutate the list, but in general I'm wary of using a mutable type as a default value. Might be best to replace it with a tuple just to be safe.